config/example-extras.cfg

```diff
@@ -791,6 +791,18 @@
 #   by during the check_gain_time check. It is rare to customize this
 #   value. The default is 2.
 
+# Tool to disable heaters when homing or probing an axis
+[homing_heaters]
+#   A comma separated list of steppers that should cause heaters to be
+#   disabled. The default is to disable heaters for any homing/probing
+#   move.
+#   Typical example: stepper_z
+steppers:
+#   A comma separated list of heaters to disable during homing/probing
+#   moves. The default is to disable all heaters.
+#   Typical example: extruder, heater_bed
+heaters:
```

Comment:
FYI, the convention of this file is to list the option name first (commented out) and then place the description of the option below it.

Reply:
Oops, that kind of slipped by me somehow, I'll correct that

Reply:
commit 31256197765cd1ad6514fd751ca5463801456a92